climada/hazard/trop_cyclone.py

```diff
@@ -151,7 +151,18 @@ def set_from_tracks(self, tracks, centroids=None, description='',
             coastal_idx = ((centroids.dist_coast < INLAND_MAX_DIST_KM * 1000)
                            & (np.abs(centroids.lat) < 61)).nonzero()[0]
 
-        LOGGER.info('Mapping %s tracks to %s centroids.', str(tracks.size),
+        # Restrict to coastal centroids within reach of any of the tracks
+        t_lon_min, t_lat_min, t_lon_max, t_lat_max = tracks.get_bounds(
+            deg_buffer=CENTR_NODE_MAX_DIST_DEG)
```

Comment:
I generally agree that more explicit naming and more variables are beneficial. But then we should change the pylint directive that restricts the number of local variables in a function:
https://github.com/CLIMADA-project/climada_python/blob/dc82cc86c026bbd90c40733c62c9474d4088dad2/.pylintrc#L402-L403
The same goes for the maximum number of lines per module:
https://github.com/CLIMADA-project/climada_python/blob/dc82cc86c026bbd90c40733c62c9474d4088dad2/.pylintrc#L218-L219

Reply:
Good point. Let's take that up Wednesday at the developer's meeting quickly?